fuzz/fuzz.go

```diff
@@ -0,0 +1,17 @@
+package fuzz
+
+import (
+	"context"
+	"github.com/google/wire/internal/wire"
+	"os"
+	"strings"
+)
+
+func Fuzz(data []byte) int {
+	ctx := context.Background()
+	_, err := wire.Load(ctx, "/", os.Environ(), strings.Split(string(data), " "))
```

Comment:
The most useful thing to fuzz is not the patterns passed to `Load`, but source code being processed and analyzed by Wire. See our internal test suite for how we're testing the tool against real Go source.